README.md

```diff
@@ -157,18 +157,60 @@ the file manually.
 
 ### Spacemacs
 
-Most probably you already have a private `org-roam` layer, if not, see examples
+From version 0.300.0, Spacemacs offers support for `org-roam`. This means private layers are no longer necessary.
+If you have already configured `org-ref`, you can follow this [blog post](https://philipperambert.com/Installing-Org-Roam-Bibtex-In-Spacemacs).
+
+Inside the function `(defun dotspacemacs/layers () ... )` uncomment the org layer and switch on roam-support. 
+You will also need to enable `bibtex`. 
+You should have something that looks like:
+```el
+(defun dotspacemacs/layers () 
+;;Your custom config goes here
+org
+(org :variables
+     org-enable-roam-support t)
+bibtex
+ )
+```
+
+Add `org-roam-bibtex` to `dotspacemacs-additional-packages`:
+```el
+dotspacemacs-additional-packages '(... org-roam-bibtex ...)
+```
+
+
+Inside the function `(defun dotspacemacs/user-config () ... )` set the custom variables and add a hook to org-mode so that org-roam-mode gets automatically enabled with the opening of an org file
```

Comment:
I'm not sure we need instructions for Org Roam in ORB's README. A link would be enough.